playbooks/besman-codeql_scan-playbook.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Author: Sandhya K
+# License: Same as Repository Licence
+# Usage : Beta-Release
+# Date : 05-Mar-2024
+
+set -e
+RED=$(tput setaf 1)
+GREEN=$(tput setaf 2)
+YELLOW=$(tput setaf 3)
+BLUE=$(tput setaf 4)
+RESET=$(tput sgr0)
+BOLD=$(tput bold)
+
+echo -e "\n${BLUE}###################################################${RESET}"
+
+echo -e "\n${BOLD}${GREEN}Info!${RESET} ${BLUE}You are using CodeQL playbook (API way)! \nSend your thoughts we would love the enhancement Ideas.\n${RESET}"
+
+echo -e "\n${BLUE}###################################################${RESET}"
+
+PROJECT_NAME=$1
+github_token=$2
+github_api_version=$3
+github_project_owner=$4
+github_repo_name=$5
+project_version=$6  
+
+# Lifecycle functions
+function __besman_init {
+
+    echo -e "\n${BLUE}🛫 Preparing your machine for Code QL Run .${RESET}"
+
+    mkdir -p $HOME/$PROJECT_NAME
+    echo -e "ls -l $PWD/$PROJECT_NAME"
+}
+
+function __besman_execute {
+    # Generating CodeQL report 
+    curl -sS \
```

Comment:
@sand-hya You should have the actual steps to perform the assessment in a steps file.

Reply:
@asa1997 will introduce the steps file and move this to steps file with modification